examples/apigee/v1beta1/envgroupattachment.yaml

```
@@ -0,0 +1,129 @@
+# SPDX-FileCopyrightText: 2024 The Crossplane Authors <https://crossplane.io>
+#
+# SPDX-License-Identifier: CC0-1.0
+
+apiVersion: apigee.gcp.upbound.io/v1beta1
+kind: EnvgroupAttachment
+metadata:
+  annotations:
+    meta.upbound.io/example-id: apigee/v1beta1/envgroupattachment
+    upjet.upbound.io/manual-intervention: "Organization requires a real Project ID"
+  labels:
+    testing.upbound.io/example-name: env-grp-attachment
+  name: env-grp-attachment
+spec:
+  forProvider:
+    envgroupIdSelector:
+      matchLabels:
+        testing.upbound.io/example-name: env-grp-attachment
+    environmentSelector:
+      matchLabels:
+        testing.upbound.io/example-name: env-grp-attachment
+
+---
+
+apiVersion: apigee.gcp.upbound.io/v1beta1
+kind: Envgroup
+metadata:
+  annotations:
+    meta.upbound.io/example-id: apigee/v1beta1/envgroupattachment
```

If we add manual intervention to a resource in the same YAML file, adding it to the dependent resources would be nice.

```suggestion
    meta.upbound.io/example-id: apigee/v1beta1/envgroupattachment
    upjet.upbound.io/manual-intervention: "This resource was skipped because the parent resource was skipped"
```

---

done